### DISCUSSION TITLE
rename workflow run name

### DISCUSSION THREAD

**Discussion by ajith29** [community/community#48360] [Enterprise]
### Select Topic Area

Question

### Body

I am trying to rename the workflow run name 
The version of github is enterprise server 3.7.
I am not able to use the dynamic run name option
i want to rename my workflow run name to say its name is pull request need to change to pull request -pull number 
how to do this with gh or curl in 

The workflow is dispatched from another workflow with the pr number so need to have the workflow run name with pull request number
![image](https://user-images.githubusercontent.com/12078835/220845664-69260c24-1510-46b2-bcea-8cecd40bafd3.png)

**Comment by jsoref** [comment]
It's a feature in [3.8](https://docs.github.com/en/enterprise-server@3.8/actions/using-workflows/workflow-syntax-for-github-actions#run-name). You'll have to wait for your GHES to upgrade. (You may be able to ask your admins to upgrade sooner...)

Please practice checking the [documentation for the version you're using](https://docs.github.com/en/enterprise-server@3.7/actions/using-workflows/workflow-syntax-for-github-actions) instead of relying on the [latest version (currently 3.8)](https://docs.github.com/en/enterprise-server@3.8/actions/using-workflows/workflow-syntax-for-github-actions#run-name) or the [github.com version of the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name).
